packages/terra-clinical-data-grid/src/terra-dev-site/doc/clinical-data-grid/DatagridExamples/CustomHeaderCellLayout.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+
+import styles from './CustomHeaderCellLayout.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  primaryText: PropTypes.string,
+  secondaryText: PropTypes.string,
+};
+
+const CustomHeaderCellLayout = ({ primaryText, secondaryText }) => (
+  <div className={cx('custom-header-cell')}>
+    <div className={cx('primary')}>
+      {primaryText}
+    </div>
+    <div className={cx('secondary')}>
```

Comment:
Do we need to check for if primary or secondary props are provided so we don't render extra divs?

Reply:
This component is luckily used within examples where there are always primary and secondary strings. We could add checks, but they'd always be truthy.